extensions/amp-carousel/amp-carousel-analytics.md

```diff
@@ -61,6 +61,14 @@ The `amp-carousel-next` event is issued when there is a travesal to the previous
 }
 ```
 
-### Vars
+## Carousel analytics variables
 
-Description of the variables can be found in the [analytics-vars.md](/extensions/amp-analytics/analytics-vars.md#fromslide) file.
+`<amp-carousel>` contributes the following variables, which can also be found in [analytics-vars.md](/extensions/amp-analytics/analytics-vars.md#fromslide).
+
+### fromSlide
```

Comment:
Amazing — @bpaduch 's changes are way better across the board :)   I'm assuming I can just delete this PR, then, but let me know if there's anything worth retaining.